src/GoogleTranslate.php

```diff
@@ -149,4 +159,17 @@ public function sanitizeLanguageCode(string $languageCode)
             get the list of valid and supported language codes by running GoogleTranslate::languages()"
         );
     }
+
+    protected function validateInput($input): void
```

Comment:
we should use the nullable type to typehint `$input` here `?string`

Reply:
As you can pass an array through to it and validate it that way (rather than using 2 different methods) it'll break when using anything with batch

Reply:
makes sense